docs/server-options.md

```diff
@@ -0,0 +1,111 @@
+---
+id: server-options
+title: Temporal server options
+sidebar_label: Start options
+---
+
+## Overview
+
+To run the Temporal, include the temporal server package `go.temporal.io/server/temporal` in your Go application. Then use it to create and start a new server:
+
+```go
+s := temporal.NewServer()
+err := s.Start()
+if err != nil{
+	log.Fatal(err)
+}
+```
+
+## Options
```

Comment:
lets try to guide the user as much as possible on what most users may need vs what a select few may need. may want to split out "options" and "advanced options"?

Reply:
I like idea - not sure exactly what "advanced" would mean in this context? Any one of these could range from simple to complex in terms of usage and consequences. I think right now, lets just make it available as a reference.

Reply:
@flossypurse  ok it was just an idea. i like being guided to "options all of you definitely want to know about" vs "options most of you can ignore"

Reply:
Added a task to revisit this - I think it will depend on the additional context we get for each of the options.